### ISSUE TITLE
New-HPOVLicense fails in 5.20

### ISSUE THREAD

**Issue by VincentBerger** [HewlettPackard/POSH-HPOneView#481]
### Actual Behavior
New-HPOVLicense fails in 5.20 with 
```The "type" field content, "LicenseV520", included in this request is not valid. Correct the content in the 'type' field of the JSON and retry the request.```
According to the API reference distributed with 5.20, it still expects type=LicenseV500
### Steps to reproduce
```
VERBOSE: [SEND-HPOVREQUEST] Prior Global Response Error Object for 'oneview52simu1.hp.local' found. Clearing.
VERBOSE: [SEND-HPOVREQUEST] Processing 'oneview52simu1.hp.local' appliance connection request. 1 of 1
VERBOSE: [SEND-HPOVREQUEST] Requested URI '/rest/licenses' to 'oneview52simu1.hp.local'
VERBOSE: [SEND-HPOVREQUEST] Body object found. Converting to JSON.
VERBOSE: [SEND-HPOVREQUEST] HTTP Method is POST. Removing 'ApplianceConnection' NoteProperty from object(s).
VERBOSE: InputObject is [PSCustomObject]. Copying...
VERBOSE: [SEND-HPOVREQUEST] Request Body: {"type":"LicenseV520","key":"YB9E AQEA H9PQ KHUZ V2B4 HWWV Y9JL KMPL B89H MZVU DXAU 2CSM GHTG L762 PGBY V7NM KJVT D5KM EFVW TSNJ KRML 476G SMT9 YGS6 SMQQ MUCF UW2L MYN7 N2QC DHKQ XJUL LUQH TU8F 3DQC SW7N VEQW V886 FE23 YWAT J8U3 2YT5 RNNV MHS3 QKTQ 688U F8A7 F8SE Z2DX RJQ6 MHPE SN9R 3UNK TVPT 74UF NGGT EHM4 \"EVAL-N3R43A_ILR N3R43A_ILR Synergy_8Gb_FC_Upgrade_License EVAL-N3R43A_ILR\""}
VERBOSE: [SEND-HPOVREQUEST] Request: POST https://oneview52simu1.hp.local/rest/licenses
VERBOSE: [SEND-HPOVREQUEST] Request Header 1: User-Agent = HPEOneview_PowerShell/5.20 (Microsoft Windows NT 10.0.14393.0)
VERBOSE: [SEND-HPOVREQUEST] Request Header 2: Content-Type = application/json
VERBOSE: [SEND-HPOVREQUEST] Request Header 3: Accept = application/json, text/html, application/xhtml+xml
VERBOSE: [SEND-HPOVREQUEST] Request Header 4: X-API-Version = 1600
VERBOSE: [SEND-HPOVREQUEST] Request Header 5: accept-language = en_US
VERBOSE: [SEND-HPOVREQUEST] Request Header 6: accept-encoding = gzip, deflate
VERBOSE: [SEND-HPOVREQUEST] Request Header 7: auth = [*****REDACTED******]
VERBOSE: [SEND-HPOVREQUEST] Net.WebException Error caught
VERBOSE: [SEND-HPOVREQUEST] Exception Object:
PSMessageDetails      :
Exception             : System.Net.WebException: The remote server returned an error: (400) Bad Request.
                           at System.Net.HttpWebRequest.GetResponse()
                           at CallSite.Target(Closure , CallSite , WebRequest )
TargetObject          :
CategoryInfo          : NotSpecified: (:) [], WebException
FullyQualifiedErrorId : WebException
ErrorDetails          :
InvocationInfo        : System.Management.Automation.InvocationInfo
ScriptStackTrace      : at Send-HPOVRequest<Process>, C:\Users\demopaq\Documents\PowerShell\Modules\hponeview.520\5.20.2422.3962\HPOneView.520.psm1: line 5610
                        at New-HPOVLicense<Process>, C:\Users\demopaq\Documents\PowerShell\Modules\hponeview.520\5.20.2422.3962\HPOneView.520.psm1: line 128819
                        at <ScriptBlock>, <No file>: line 1
PipelineIterationInfo : {}



VERBOSE: [SEND-HPOVREQUEST] Exception Message: The remote server returned an error: (400) Bad Request.
VERBOSE: [SEND-HPOVREQUEST] InnerException FullyQualifiedErrorId:
VERBOSE: [SEND-HPOVREQUEST] InnerException Message:
VERBOSE: [SEND-HPOVREQUEST] Exception
VERBOSE: [SEND-HPOVREQUEST] Getting Error Response
VERBOSE: [SEND-HPOVREQUEST] ERROR RESPONSE: {"errorCode":"INVALID_JSON_TYPE","message":"Invalid JSON type.","details":"The \"type\" field content, \"LicenseV520\", included in this request is not valid.","messageParameters":[],"recommendedActions":["Correct the content in the 'type' field of the JSON and retry the request."],"errorSource":null,"nestedErrors":[],"data":{}}
VERBOSE: [SEND-HPOVREQUEST] Response Status: HTTP 400 [Bad Request]
VERBOSE: [SEND-HPOVREQUEST] Response Header: Date = close
VERBOSE: [SEND-HPOVREQUEST] Response Header: Server = chunked
VERBOSE: [SEND-HPOVREQUEST] Response Header: Cache-Control = application/json; charset=utf-8
VERBOSE: [SEND-HPOVREQUEST] Response Header: Pragma = 0
VERBOSE: [SEND-HPOVREQUEST] Response Header: X-XSS-Protection =
VERBOSE: [SEND-HPOVREQUEST] Response Header: X-Frame-Options =
VERBOSE: [SEND-HPOVREQUEST] Response Header: X-Content-Type-Options =
VERBOSE: [SEND-HPOVREQUEST] Response Header: Connection =
VERBOSE: [SEND-HPOVREQUEST] Response Header: Transfer-Encoding =
VERBOSE: [SEND-HPOVREQUEST] Response Header: Content-Type =
VERBOSE: [SEND-HPOVREQUEST] Response Header: Expires =
VERBOSE: [SEND-HPOVREQUEST] HTTP 400 error caught.
VERBOSE: [NEW-ERRORRECORD] Building ErrorRecord object
VERBOSE: [SEND-HPOVREQUEST] Cleaning up HttpWebRequest
VERBOSE: [NEW-ERRORRECORD] Building ErrorRecord object
New-HPOVLicense: The "type" field content, "LicenseV520", included in this request is not valid. Correct the content in the 'type' field of the JSON and retry the request.
```

### Version Information

```PowerShell
oneview52simu1.hp.local             LibraryVersion Path
-----------------------             -------------- ----
ApplianceVersion: 5.20.00.419867.00 5.20.2422.3962 C:\Users\demopaq\Documents\PowerShell\Modules\hponeview.520\5.20.24…
```

**Comment by ChrisLynchHPE** [comment]
This is a known issue and is addressed in the next library update.

**Comment by VincentBerger** [comment]
Fix verified in 5.20.2452. Closing
